calculus/source/03-AD/08.ptx

```diff
@@ -12,30 +12,96 @@
   <subsection>
     <title>Activities</title>
     
-      <activity xml:id="activity-ad8-box"> <title>The box</title>
+      <activity xml:id="activity-ad8-box"> 
+        <title>The box</title>
         <introduction>
             <p>
-              Help your company design an open box (no lid) with maximum volume given the following constraints:
+              Help your company design an open box (that is, a box with no lid) with maximum volume given the following constraints:
                 <ul>
-                <li>The box must be made from the following material: an 8 by 8 inches piece of cardboard.</li>
-                <li>To create the box, you are asked to cut out a square from each corner of the 8 by 8 inches piece of cardboard and to fold up the flaps to create the sides.</li>
+                <li>The box must be made from a square piece of cardboard that is 8 inches on each side.</li>
+                <li>To create the box, you are asked to cut out smaller squares from each corner of the 8 by 8 inches piece of cardboard and to 
+                  fold up the flaps to form the sides.</li>
                 </ul>
             </p>
         </introduction>
         <task>
-            <p>Draw a diagram illustrating how the box is created.</p>
+            <statement>
+              <p>Draw a diagram illustrating how the box is created. Label the lengths of segments in the diagram and write <m>x</m> for the
+              sides of the smaller squares to be removed from the 8 by 8 square.</p>
+            </statement>
+          <solution>
+            <image source="thebox-1">
+              <shortdescription>Square of side length 8 with square corners of side length <m>x</m> removed</shortdescription>
+            </image>
+          </solution>
```

Comment:
For the purpose of a 4S in-class activity, I think providing four choices of diagrams and letting students choose which is most appropriate is better scaffolded, especially this early in a section.